core/egov-localization/src/main/java/org/egov/persistence/repository/MessageRepository.java

```diff
@@ -83,7 +82,7 @@ public int getBatchSize() {
 		}
 		logger.info("Status of Insert is : " + values.length);
 		if(values.length > 0){
-			Tenant tenant = new Tenant(createMessagesRequest.getTenantId());
+			// Tenant tenant = new Tenant(tenantId);
 			// messageCacheRepository.bustCacheEntry(createMessagesRequest.getLocale(), tenant);
```

Comment:
As I do not have Redis in my local, to avoid the errors, I've commented this. 
Once the code has been pushed to masters, I will removed the comment and test it right away